crates/forge-runner/src/running.rs

```diff
@@ -281,11 +282,9 @@ pub fn run_test_case(
     );
 
     let execution_resources = get_all_execution_resources(&forge_runtime);
+    let block_context = &get_context(&forge_runtime).block_context.clone();
```

Comment:
```suggestion
    let block_context = get_context(&forge_runtime).block_context;
```
Could it not be just that and then pass it with reference?